ml/dgl_utils.py

```diff
@@ -0,0 +1,280 @@
+# classes for converting a dataframe or Graphistry Plottable into a DGL
+import numpy as np
+import pandas as pd
+import torch
+
+from ml import constants as config
+from ml.umap_utils import baseUmap, umap_kwargs_euclidean
+from ml.utils import pandas_to_dgl_graph, process_dirty_dataframes, setup_logger
+
+logger = setup_logger(__name__)
+
+
+def get_vectorizer(name):
+    if type(name) != str:
+        logger.info(f"Returning {type(name)} vectorizer")
+        return name
+    if name == config.DIRTY_CAT:
+        return process_dirty_dataframes
+    if name == config.SKLEARN:
+        # TODO
+        return
+    logging.warning(
+        f"Vectorizer name must be one of [{config.DIRTY_CAT}, {config.SKLEARN}, ..]"
+    )
+    return
+
+
+def get_dataframe_for_target(target, df, name):
```

Comment:
not sure what `target` is so unsure of useful generalness here. ok to be minimal, esp. initially...